.gitignore

```diff
@@ -41,3 +41,7 @@ node_modules
 
 # Code coverage
 ./coverage
+
+# dotenv
+.env
```

Comment:
Can't you replace these two with?
```js
.env*
```

Reply:
By the way I wrote the `config` there's no need of having the `.env.test` file